### PR TITLE
Create a card box-shadow effect to be to able distinguish between the cards. #2906

### LINKED ISSUE #2906: Produce a Card-box-shadow effect
Describe the problem
In the "How to Support?" section of the "Contribute" Webpage, all the three cards of Student, Teacher and Developer doesn't look distinguished from each other and is being visible as a single block.

To Reproduce

1. Go to [https://circuitverse.org/contribute]
2. Scroll down to the "How to Support Section"
3. See error

Expected behavior
To produce a card box-shadow effect to distinguish the cards from each other.

Screenshots
Before:
CircuitVerse-How to support(OLD)

After:
CircuitVerse-How to support(NEW)

Are you working on this issue? (Yes/No)
Yes

### PR DESCRIPTION
**Pull request by bhutani2002**
Fixes #2906 
Describe the problem
In the "How to Support?" section of the "Contribute" Webpage, all the three cards of Student, Teacher and Developer doesn't look distinguished from each other and is being visible as a single block.

To Reproduce

Go to [https://circuitverse.org/contribute]
Scroll down to the "How to Support Section"
See error
Expected behavior
To produce a card box-shadow effect to distinguish the cards from each other.

Screenshots
Before:
CircuitVerse-How to support(OLD)
CircuitVerse-Contribute-Card-Old (2)

After:
CircuitVerse-How to support(NEW)
CircuitVerse-Contribute-Card-New (1)

### CODE REVIEW

**Comment by Prerna-0202** [comment]
Can you please attach the screenshots and link the issue with pr ?

**Comment by bhutani2002** [comment]
> Fix Code Climate issues and Can you please attach the screenshots and link the issue with pr ?

Can you please elaborate on this?? I am not able to get ,what it's trying to say.
image

**Comment by Prerna-0202** [comment]
> > Fix Code Climate issues and Can you please attach the screenshots and link the issue with pr ?
> 
> Can you please elaborate on this?? I am not able to get , what it's trying to say. image

Yaa, it is saying that you have to leave an empty line after the declaration  (after }  this bracket )...

**Comment by bhutani2002** [comment]
Hi @Prerna-0202 thanks for your replies. I guess finally, there no issues. Please check it once. It happened after I guess 8 pushes just due to that spacings and new lines thing 😂. Thank You

**Comment by bhutani2002** [comment]
Hey @Prerna-0202 what's the status of merging?
Thank You.

**Comment by bhutani2002** [comment]
Hey @gr455 , @tachyons can you please look into the merging process.
Thank You

File: config/puma.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
```
Comment (gr455) [inline config/puma.rb:24]: Everything else looks good. Just remove this change. Thanks :+1:

**Review by vedant-jain03 — APPROVED** [review]

File: config/puma.rb (hunk 1 above)
Comment (bhutani2002) [reply config/puma.rb]: Done. Thank You

**Pull request merged**